### DISCUSSION TITLE
Nvm recognized by gitbash but not cmd or powershell on win11

### DISCUSSION THREAD

**Discussion by Avi-E-Koenig** [coreybutler/nvm-windows#1182] [General]
I have an issue with nvm. I installed nvm-windows. It was installed with the default settings. I ensured that the environment variables were properly recognized. 

In Git Bash nvm, npm node are properly recognized.

 In the beginning, npm was not operating properly, but then I set up npm to use the Git Bash terminal as default, and then it started working properly. 

However, when I go to PowerShell or CMD, nvm,npm,node are not recognized.

**Comment by coreybutler** [comment]
Run `nvm debug` in git bash and post the results here.
